src/shipit_signoff/shipit_signoff/api.py

```diff
@@ -253,6 +252,10 @@ def delete_signature(uid):
         logger.error(
             'User %s attempting to remove missing signature from step %s', email, uid)
         abort(409, str(e))
+    except NoChangingCompletedPolicyError as e:
+        logger.error(
```

Comment:
All levels of logs (``error``, ``warning``, ``info``, ``debug``) are sent to Papertrails. But only levels ``error`` and ``warning`` are sent to Sentry.

I was thinking of writing down the policy for Sentry in some form like this:
staging:
 - there should be no errors when a deployment to staging happens
production:
 - errors reported to Sentry need to be looked at ASAP and a responsible person will be notified to look at it.
 - warnings can be used to collect certain undesirable behaivour of your code (eg. how many users fail to complete login form, because of specific reason)

Maybe a better level for this would be ``warning``, eg ``logger.warn('...')``? What do you think? Does the above "policy" make sense?